src/org/zaproxy/zap/extension/ascanrulesBeta/InsecureHTTPMethod.java

```diff
@@ -245,7 +279,7 @@ public void scan() {
 				//rely on the OPTIONS METHOD, but potentially verify the results, depending on the Threshold.
 				for (String enabledmethod: allowedmethods.toUpperCase().split(",")) {
 					enabledmethod = enabledmethod.trim();  //strip off any leading spaces (it happens!)
-					
+									
```

Comment:
This change should be reverted.
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting